### ISSUE TITLE
Make search input more accessible

### ISSUE THREAD

**Issue by 3y3** [diplodoc-platform/components#216]
Search button into search input doesn't have relevant role
Search input doesn't have state. Looks like we need to add [expanded](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded) state
